editor/src/components/canvas/canvas-types.ts

```diff
@@ -22,7 +22,7 @@ import { xor } from '../../core/shared/utils'
 
 export const CanvasContainerID = 'canvas-container'
 
-export const enum CSSCursor {
+export enum CSSCursor {
```

Comment:
`const enum`s are a special thing in typescript and are not supported by babel-typescript. We don't really care about the difference so I changed them all.

Reply:
There's an explanation of them here: https://www.typescriptlang.org/docs/handbook/enums.html#const-enums
TLDR - they are completely removed during compilation, so their only benefit is to slightly reduce the amount of output code.